src/extensions/cite/index.ts

```diff
@@ -10,23 +61,46 @@ import { ready, create, replace, select, attr } from '../../util'
  * </span>
  * ```
  *
- * So that we can style the date, a better encoding would simply be:
+ * A better encoding for styling, and concision, is:
  *
  * ```html
- * <time datetime="2019-08-23" itemscope="" itemtype="http://schema.org/Date" itemprop="datePublished" >2019-08-23</time>
+ * <time datetime="2019-08-23" itemprop="datePublished" >2019-08-23</time>
  * ```
  *
  * Noting that for `<time>` elements the `datetime` attribute is used as the property value:
- * https://www.w3.org/TR/microdata/#values
+ * https://www.w3.org/TR/microdata/#values and it should not have `itemscope` and `itemtype`
+ * attributes.
  */
 ready(() =>
-  select('span > :--datePublished').forEach(meta => {
-    const date = attr(meta, 'content') as string
+  select(':--references :--reference').forEach(reference => {
+    // Change `authors` property from list to nested spans
+    select(reference, 'ol:--authors').forEach(elem => {
+      select(reference, 'li:--author').forEach(elem => tag(elem, 'span'))
+      return tag(elem, 'span')
+    })
```

Comment:
👍 for `no-shadow`.

Reply:
I felt that in this context, authors as part of a `references` item, it would be less hassle for CSS authors e.g implementing different citation styles to not have to turn off the default styling of lists. But if that's not an issue, I'm happy to go back to the original proposal.

Linting addressed in https://github.com/stencila/thema/pull/84/commits/8c7dc1e751c54dc6f6154a7b39a9a569f2ea8d0e